### ISSUE TITLE
handle_model() drops other language values when serializing I18N fields

### ISSUE THREAD

**Issue by jacobtylerwalls** [archesproject/arches#11315]
`JSONSerializer.handle_model()` drops other language values besides the active language's when serializing, which can lead to data loss.

***

> Sure, my experience was that when serializing, only the active language's value got serialized, which is a problem if that is then deserialized somewhere else (accepted from an AJAX request, or a roundtrip we're doing ourselves in [resource_graphs/exporter.py](https://github.com/archesproject/arches/pull/11096/files#diff-4ef90fb853af3f36fd77af185226287e27a047f38c12c781dcb513f655d658eb).

_Originally posted by @jacobtylerwalls in https://github.com/archesproject/arches/pull/11096#discussion_r1707895774_

**Comment by jacobtylerwalls** [comment]
To reproduce:

- Edit an I18n_TextField field such as ResourceInstanceLifecycle.name in the django admin to have additional language values
- Export graph
- Inspect json: notice only one language is persisted

Throughout the system we hydrate from serialized graphs, so this will lead to data loss.

**Comment by chrabyrd** [comment]
I'm unsure that hydrating from serialized graphs will result in data loss. I believe this is already handled for in the graph_x_published_graphs table by creating a serialized graph for each enabled language

**Comment by jacobtylerwalls** [comment]
I'm not 100% sure of my footing here either, but @johnatawnclementawn and I noticed this morning that these fields are both I18n_TextFields:

- Card.name
- ResourceInstanceLifecycle.name

and yet they serialize differently to graph json:

```json
"graph": [
        {
            "cards": [
                {
                    "name": {
                        "en": "String node"
                    },
            },
            "resource_instance_lifecycle": {
                "name": "Standard",
            },
```

which is what I was alluding to here:

> my experience was that when serializing, only the active language's value got serialized, which is a problem if that is then deserialized somewhere else (accepted from an AJAX request, or a roundtrip we're doing ourselves in [resource_graphs/exporter.py](https://github.com/archesproject/arches/pull/11096/files#diff-4ef90fb853af3f36fd77af185226287e27a047f38c12c781dcb513f655d658eb).

***
I take your point that published graphs are supposed to have some accounting for this, so if this less about data loss and more about consistency then no harm done, I only feel it's still worth looking into -- as a developer, I'm not totally sure what's supposed to happen.

**Comment by chrabyrd** [comment]
Yeah no this is fair. Arguably I'd say it'd be _better_ to handle for this by having proper i18n data for everything. Then we could stop making so many graph_x_published_graph records and have a single source of truth for all languages for published graphs

**Comment by johnatawnclementawn** [comment]
It seems like there are two issues here:
1. As Jacob [points out](https://github.com/archesproject/arches/issues/11315#issuecomment-2920329699), resource_instance_lifecycle names are not exported not exported with raw i18n json. This is likely because there is not a method on the serialized graph like [get_widgets](https://github.com/archesproject/arches/blob/ea78b83e5d16d25b3e0ca182f1b7f2dd89388d45/arches/app/models/graph.py#L1886) which is [called during serialize](https://github.com/archesproject/arches/blob/ea78b83e5d16d25b3e0ca182f1b7f2dd89388d45/arches/app/models/graph.py#L1959-L1965) that fetches the raw_i18n_json for resource_instance_lifecycle names.
2. The second, overarching issue is ~~that even though we are asking for raw i18n json for i18n_TextFields on the models that comprise the serialized graph, only the k/v for the active language is being returned - which is a regression in v8 compared to 7.6 confirmed by Cyrus~~ 
    a. I think this is actually not a regression with the serializer, but more an issue with _**when**_ we are serializing the raw json or just the active language. Last week when sniffing around this issue because of this [PR](https://github.com/archesproject/arches-lingo/pull/312), I found that I could edit a card's description in two different languages and both languages would show up in the db on the draft graph, but if I published, only the value for the active language would be saved. To me this indicates that when we're constructing the serialized graph during publication, we're not passing `use_raw_i18n_json` somewhere we should.

**Comment by johnatawnclementawn** [comment]
Surface area for #12269 is small - ensure just lifecycle states aren't dropping non-active language k/v pairs for i18n_textfields during export. It seems like #12182 does a good job to ensure this doesn't happen when serializing the graph in general (e.g. during promoting a draft graph to current graph), but we should still keep an eye out for data-loss
